src/global_logic.js

```diff
@@ -2816,6 +2800,24 @@ module.exports.treatSupportAbility = function (totals, chara, buff) {
                         totals[key]["normalBuff"] += support.value;
                     }
                     continue;
+                case "hanged_man_reversed":
+                    if (index > 4) {
+                        if (epic.isAllUniqueArm(arml, comb)) {
+                            if (totals[key].isConsideredInAverage) {
+                                for (var key2 in totals) {
+                                    totals[key]["caimOther"] += support.value;
+                                    totals[key]["normalDamageLimit"] += support.value;
+                                    totals[key]["ougiDamageLimit"] += support.value;
+                                }
+                            } else {
```

Comment:
support_filter module now provides iteration support_filter.range.element["earth"]
no more duplicated code for non-isConsideredAverage
(the iteration function return self if isConsideredInAverage was off)

```js
for (let [name, chara] of range.element["earth"](totals, key)) {
    chara["caimOther"] += support.value;
    chara["normalDamageLimit"] += support.value;
    chara["ougiDamageLimit"] += support.value;
}
```

EDIT: "chara" was used in outer scope, "let" protect with in the scope though,
another variable name for avoid the confuse.

Reply:
It is set up how other skills are set up. I am not doing let and const stuff which behaves differently from other stuff.

Reply:
they are an old code. the improve points are
- deep unnecessary indent
- duplicated code for non-isConsideredInAverage  (it make double maintenance cost)

```js
if (index > 4 && epic.isAllUniqueArm(arml, comb)) {
    for (let [name, chara] of range.element["earth"](totals, key)) {
        chara["caimOther"] += support.value;
        chara["normalDamageLimit"] += support.value;
        chara["ougiDamageLimit"] += support.value;
    }
}
```